.github/workflows/ipc_fuzzer.yml

```diff
@@ -57,10 +57,24 @@ jobs:
           sudo dpkg --add-architecture i386
           sudo apt-get update
 
-      - name: apt-get install
-        run: sudo apt-get update && sudo apt-get -y install
-          clang llvm libc6-dev-i386 libstdc++-12-dev:i386
-          ninja-build device-tree-compiler python3-pyelftools
+      - name: apt-get install i386 packages
+        run: |
+
+          # We don't know why Github requires us to explicit all these
+          # i386 dependencies. On a vanilla Ubuntu 22.04 the last
+          # apt-get is enough.
+
+          sudo apt-get -y install libgcc-s1:i386 libc6-dev:i386 libstdc++6:i386
+          sudo apt-get -y install libgomp1:i386  libitm1:i386 libatomic1:i386
+          sudo apt-get -y install libasan8:i386 libubsan1:i386 libquadmath0:i386
+
+          sudo apt-get -y install \
+             libc6-dev-i386 libc6-dev:i386 libstdc++-12-dev:i386
```

Comment:
So, indent of three here, two in many places, and one before clang?  

Got it.  Approved.